test/JDBC/expected/BABEL-DATABASEPROPERTYEX.out

```diff
@@ -3,7 +3,7 @@ select databasepropertyex(N'master',N'Collation')
 GO
 ~~START~~
 sql_variant
-sql_latin1_general_cp1_ci_as
+bbf_unicode_cp1_ci_as
```

Comment:
Is this right ? Should databasepropertyex be returning the original collation & not the internal bbf mapped collation.

Reply:
Yes this is fine. Because we store the bbf collation in all  catalogs. Ideally, user should see the bbf collation.